windows-installer/template.nsi

```diff
@@ -19,6 +19,17 @@ InstallDir "$PROGRAMFILES\Certbot"
  
 SetCompressor lzma
 
+Function .onInit
+  ; Check that Powershell is at least 5.0.
+  nsExec::ExecToStack `powershell -ExecutionPolicy RemoteSigned -Command "Write-Host -NoNewline (Get-Host | Select-Object Version).Version.CompareTo([System.Version]'5.0')"`
```

Comment:
As I said in the previous PR, the two approaches a very similar, and it is quite difficult to choose.

However here using `$PSVersionTable.PSVersion` would complexify the code, because the interpolation of `$VARIABLE` between NSIS and the underlying powershell code would need to be taken care of, and the comparison with the straight forward invocation of `[System.Version]'5.0'` is not possible anymore.

So I prefer to stay with `(Get-Host | Select-Object Version)` here.